modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4408,11 +4409,13 @@ cp-compare-content-mini-display-context-help=Provides helper methods to retrieve
 cp-content-list-renderer-key=Commerce Product Content List Renderer Key
 cp-definition=Commerce Product
 cp-definition-help=Represents the commerce product definition that is displayed in the virtual product detail.
+cp-definition-id=Represents the ID of commerce product definition that is displayed.
```

Comment:
This is wrong, please follow the standard pattern or delete it and use an existing key

Reply:
This is not something we can show to the citizen dev, please change it with something readable.